Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2-slim
+RUN groupadd user && useradd --home /code --disabled-password --gecos "" -g user user
+
+COPY "feedfetcher.py","rssfeed.py","settings.py.docker" /code

```

Comment:
I wanted to avoid sending the whole directory inside the image, but I've done it differently.